ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1481,6 +1492,54 @@ namespace OrdinalAxis {
             return ret;
         }
 
+        /**
+         * If overscroll is pixel or pecentage value, convert it to axis range.
+         *
+         * @private
+         * @param {number | string} overscroll
+         * Overscroll value in axis range, pixels or percentage value.
+         * @return {number}
+         * Overscroll value in axis range.
+         */
+        public convertOverscroll(overscroll : number | string = 0) : number {
```

Comment:
I'm curious about the placement of this method. It is not really part of the Ordinal Axis. I don't want to pollute Core Axis with Stock code but we don't have a separate StockAxis extension yet. Perhaps it is time to extract Overscroll option to a new file? What do you think @TorsteinHonsi

Reply:
Sure, if you prefer so. And I agree, if the `overscroll` feature also works with a non-ordinal axis, it strictly doesn't belong in the OrdinalAxis.ts file.

Reply:
To avoid further delay, we decided to keep it here for now. Will be taken care of when working on other related issues/bug/feature.